scripts/update-endpoints/typescript.js

```diff
@@ -149,6 +149,9 @@ async function run() {
           "For https://api.github.com, set `baseUrl` to `https://uploads.github.com`. For GitHub Enterprise Server, set it to `<your hostname>/api/uploads`",
         required: true,
       });
+
+      const data = parameters.find((parameter) => parameter.name === "data");
+      data.type = "string | Buffer";
```

Comment:
That's great, thank you!

Can you also update the comment in line 143? The comment could just say

```js
// workarounds for "Upload a release asset"
```

and then add separate comments above the two respective workarounds explaining what is happening

Reply:
Most definitely, I'll add those in.